### DISCUSSION TITLE
An idea on how to fix conversation continuity issues

### DISCUSSION THREAD

**Discussion by eddietoth** [chatwoot/chatwoot#8447] [Feature Requests]
### Is your feature or enhancement related to a problem? Please describe.

My business doesn't have a client interface or anything like that where my clients have an account, so I can't really verify that I'm talking with my clients. Below, I've described a possible solution on how to fix the conversation continuity issues, and make it easy to verify that I'm talking with my actual clients. 

### Describe the solution you'd like

![Screenshot 2023-11-23 181729](https://github.com/chatwoot/chatwoot/assets/121629496/f257f75a-48c7-474c-ba6d-ad7d6060e150)
![Screenshot 2023-11-23 181724](https://github.com/chatwoot/chatwoot/assets/121629496/4303630a-8063-46d0-b783-637ba4a9a51f)
These are some **amazing** examples of how this could be implemented in chatwoot!

**Comment by pranavrajs** [comment]
Moving to this to discussions we don't have enough information on this issue. @eddietoth1 Are you looking to implement a verification before the user can start the conversation?
